testing/chain.go

```diff
@@ -85,6 +85,7 @@ var (
 type TestChain struct {
 	t *testing.T
 
+	Coordinator   *Coordinator
```

Comment:
chains now have a reference to the coordinator where they can get the current time from